docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -639,6 +639,25 @@ How to **investigate**:
       - If you encounter any Compactor resource issues, add CPU/Memory as needed temporarily, then scale back later.
       - You can also optionally scale replicas and shards further to split the work up into even smaller pieces until the situation has recovered.
 
+### MimirCompactorHasRunOutOfDiskSpace
+
+This alert fires when the compactor has run out of disk space at least once.
+When this happens the compaction will fail and after some time the compactor will retry the failed compaction.
+Unless the compactor concurrency is >=1 it is very likely that on each retry it will just hit the same disk space limit again and it won't be able to recover on its own.
```

Comment:
I would perhaps try to explain it:

```suggestion
It is very likely that on each retry of the job, compactor will just hit the same disk space limit again and it won't be able to recover on its own. If compactor concurrency is higher than 1, then it could have been just an unlucky combination of jobs that caused compactor to run out of disk space.
```